easybuild/framework/easyconfig/easyconfig.py

```diff
@@ -1252,6 +1252,8 @@ def process_easyconfig(path, build_specs=None, validate=True, parse_only=False,
                 dep = ec.toolchain.as_dict()
                 _log.debug("Adding toolchain %s as dependency for app %s." % (dep, name))
                 easyconfig['dependencies'].append(dep)
+                if dep['hidden']:
+                    easyconfig['hiddendependencies'].append(dep)
```

Comment:
No, I checked where `hiddendependencies` is used, and it's not used to determine the module name to use.

I dropped this in https://github.com/geimer/easybuild-framework/pull/9, I'm fairly sure it's not needed at all.